Userland/Libraries/LibGfx/AntiAliasingPainter.cpp

```diff
@@ -18,125 +18,19 @@
 
 namespace Gfx {
 
-void AntiAliasingPainter::draw_anti_aliased_line(FloatPoint actual_from, FloatPoint actual_to, Color color, float thickness, LineStyle style, Color, LineLengthMode line_length_mode)
+void AntiAliasingPainter::draw_anti_aliased_line(FloatPoint actual_from, FloatPoint actual_to, Color color, float thickness, LineStyle style, Color)
 {
     // FIXME: Implement this :P
     VERIFY(style == LineStyle::Solid);
```

Comment:
(This should be fairly easy now if we actually need it – just pass `.dash_pattern` for StrokeStyle, and set caps to round or not depending on dots or dashed. See LibPDF's Renderer.cpp `Renderer::show_clipping_paths()` for an example.)

Reply:
> if we actually need it 

The other styles are used in LibWeb for borders (but probably not anywhere else) -- so it's debatable, but it's less code either way